src/tools/DemandGenerator.cpp

```diff
@@ -74,8 +79,8 @@ Demand * DemandGenerator::generateSinglePerturbatedDemand(bool checkFeasibility)
 	// number of demands in the history
 	int nbPastDemands = demandHistory_.size();
 
-	// number of demands generated until now
-	Demand * pGeneratedDemand;
+	// // number of demands generated until now
```

Comment:
Delete?

Reply:
I am not sure to want to change that. I have some current errors with the dynamic version as I was running the  static one ...  I/You need to dive in, as there are some refactoring to do. Issue, cleaning dynamic version ?